transformers/hf_api.py

```diff
@@ -0,0 +1,152 @@
+# coding=utf-8
+# Copyright 2019-present, the HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function
+
+from typing import List, NamedTuple
+import os
+from os.path import expanduser
+
+import requests
+from requests.exceptions import HTTPError
+
+ENDPOINT = "https://huggingface.co"
+
+class S3Obj:
+    def __init__(self, filename: str, LastModified: str, ETag: str, Size: int):
```

Comment:
Some property are CamelCased and some are snake_case, is it on purpose ?

Reply:
Yeah that's basically the format on the S3 response field that I just proxy through the API

Reply:
Yeah that's basically the format on the S3 response field that I just proxy through the API